src/Submissions/Api/Areas/Biobank/Controllers/ProfileController.cs

```diff
@@ -56,11 +55,11 @@ public class ProfileController : Controller
     private readonly PublicationsOptions _publicationsConfig;
     private readonly UserManager<ApplicationUser> _userManager;
     private readonly SignInManager<ApplicationUser> _signInManager;
+  private readonly ILogoService _logoService;
```

Comment:
```suggestion
    private readonly ILogoService _logoService;
```
(indentation consistent)